src/lib/components/singletons/ShareUrl.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+	import { writable } from 'svelte/store';
+	import Icon from '@iconify/svelte';
+
+	let currentUrl: string = '';
+	let isCopied = writable(false);
+
+	onMount(() => {
+		currentUrl = window.location.href;
```

Comment:
HI @grmbyrn, I guess there must also be an option in SvelteKit to get the current URL. For example:

```
<script lang="ts">
	import { onMount } from 'svelte';
	import { writable } from 'svelte/store';
	import Icon from '@iconify/svelte';
	import { page } from '$app/stores'; // <---- THIS

	let currentUrl: string = '';
	let isCopied = writable(false);
	
	function copyUrlToClipboard() {
		const textarea = document.createElement('textarea');
		textarea.value = $page.url.pathname; // <---- AND THIS
		document.body.appendChild(textarea);

		textarea.select();
		textarea.setSelectionRange(0, 99999);

		document.execCommand('copy');

		document.body.removeChild(textarea);

		isCopied.set(true);

		setTimeout(() => {
			isCopied.set(false);
		}, 3000);
	}
</script>
```

Reply:
Hi @josecelano I also used `$page.url.pathname` here, as well as `navigator.clipboard` as it's consistent with what's used in the `CodeBlock` component.